pkg/model/file/interfaces.go

```diff
@@ -40,7 +40,7 @@ type Template interface {
 	Builder
 	// GetBody returns the template body
 	GetBody() string
-	// SetTemplateDefaults returns the TemplateMixin for creating a scaffold file
+	// SetTemplateDefaults sets the default values for templates
```

Comment:
```suggestion
	// SetTemplateDefaults will return an error if not be possible to set the default values for templates
```
WDYT about something as ^?

Reply:
IMHO that doesn't add any information as returning an `error` is part of the Go language.

Reply:
The comment is to describe what the func does. So, I understand that `it returns an error if not be possible to set the default values for templates`. Am I right? In this way, I did not get your comment.